### PR TITLE
Bump source deps

### PR DESCRIPTION
**Pull request by joecowton1**
## Accessibility

-   [ ] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [ ] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [ ] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -91,18 +91,19 @@
...
+        "@emotion/react": "11.1.5",
         "@guardian/ab-core": "^2.0.0",
-        "@guardian/commercial-core": "^4.25.0",
-        "@guardian/consent-management-platform": "^10",
-        "@guardian/eslint-plugin-source-foundations": "^4.0.2",
-        "@guardian/eslint-plugin-source-react-components": "^4.1.0",
-        "@guardian/libs": "^9.0.1",
-        "@guardian/source-foundations": "^4.0.2",
-        "@guardian/source-react-components": "^4.0.1",
+        "@guardian/commercial-core": "4.25.0",
```
Comment (arelra) [inline package.json:100]: Are we intentionally fixing the versions or did we accidentally drop the `^`?

File: package.json [hunk 2]
```diff
@@ -121,31 +122,37 @@
...
+        "typescript": "4.8.4",
+        "web-vitals": "^2.1.4",
+        "webpack": "^4.0.0"
     },
     "peerDependencies": {
         "@emotion/react": "^11.1.5",
-        "@guardian/consent-management-platform": "^10",
-        "@guardian/source-foundations": "^4.0.2",
-        "@guardian/source-react-components": "^4.0.1",
+        "@guardian/commercial-core": "^4.24.0",
```
Comment (arelra) [inline package.json:147]: Should be `^4.25.0`?

**Comment by arelra** [comment]
Thanks for sorting this!

Looks like there are some valid chromatic diffs.

I noticed the YoutubeAtom overlay stories are now showing a different font?

File: package.json (hunk 1 above)
Comment (joecowton1) [reply package.json:100]: @arelra fixed in https://github.com/guardian/atoms-rendering/pull/491

File: package.json (hunk 2 above)
Comment (joecowton1) [reply package.json:147]: @arelra fixed in https://github.com/guardian/atoms-rendering/pull/491

**Comment by joecowton1** [comment]
> Thanks for sorting this!
> 
> Looks like there are some valid chromatic diffs.
> 
> I noticed the YoutubeAtom overlay stories are now showing a different font?

@arelra I've moved discussion to here as this was a weird duplicate PR, sorry!
https://github.com/guardian/atoms-rendering/pull/491

I'm not seeing a different font on the youtube atom though, just text sans. What do you see?

**Pull request closed without merging**